<launch>


  <!-- custom baxter params -->
  <param name="/robot_description"
         command="$(find xacro)/xacro.py $(find jsk_2015_05_baxter_apc)/urdf/baxter.xacro" />
  <node name="robot_state_publisher"
        pkg="robot_state_publisher" type="robot_state_publisher">
    <remap from="joint_states" to="/robot/joint_states"/>
  </node>
  <rosparam param="/baxter_joint_trajectory"
            file="$(find jsk_2015_05_baxter_apc)/config/baxter_joint_trajectory.yaml"/>


  <!-- vacuum gripper -->
  <include file="$(find jsk_2015_05_baxter_apc)/launch/include/vacuum_gripper.launch" />


  <!-- baxter startup -->
  <include file="$(find jsk_baxter_startup)/baxter.launch">
    <arg name="launch_servo" value="true"/>
    <arg name="launch_joint_trajectory" value="true"/>
    <arg name="launch_gripper_action" value="false"/>
    <arg name="launch_openni" value="false"/>
    <arg name="launch_kinect2" value="false"/>
    <arg name="launch_voice_echo" value="true"/>
    <arg name="launch_moveit" value="false"/>
    <arg name="launch_teleop" value="false"/>
    <arg name="launch_tweet" value="true"/>
    <arg name="launch_mongodb" value="false"/>
    <arg name="launch_wrench" value="false"/>
    <arg name="launch_time_signal" value="false"/>
    <arg name="start_openni" value="false"/>
    <arg name="USER_NAME" value="false"/>
  </include>


  <include file="$(find jsk_2015_05_baxter_apc)/launch/include/kiva_pod_state.launch" />
  <node name="interactive_adjust_kiva_pod"
        pkg="jsk_interactive_marker" type="marker_6dof">
    <rosparam command="load" file="$(find jsk_2015_05_baxter_apc)/config/kiva_pod_interactive_marker.yaml" />
  </node>


</launch>
